<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>networkSWidget</class>
 <widget class="QWidget" name="networkSWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>335</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>317</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>300</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>130</height>
    </size>
   </property>
   <property name="title" >
    <string>Proxy server</string>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QCheckBox" name="useProxyCheckBox" >
      <property name="text" >
       <string>Use proxy server</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QFrame" name="frame" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="3" >
        <widget class="QLineEdit" name="passLineEdit" >
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Password:</string>
         </property>
         <property name="buddy" >
          <cstring>passLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="userLineEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>User:</string>
         </property>
         <property name="buddy" >
          <cstring>userLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLineEdit" name="portLineEdit" />
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Port:</string>
         </property>
         <property name="buddy" >
          <cstring>portLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="hostLineEdit" />
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Host:</string>
         </property>
         <property name="buddy" >
          <cstring>hostLineEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>145</y>
     <width>317</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title" >
    <string>File transfer</string>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="2" column="1" >
     <widget class="QSpinBox" name="sendPortSpinBox" >
      <property name="buttonSymbols" >
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="maximum" >
       <number>65535</number>
      </property>
      <property name="minimum" >
       <number>1</number>
      </property>
      <property name="value" >
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2" >
     <widget class="QPushButton" name="testPortPushButton" >
      <property name="text" >
       <string>Test port</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" >
     <widget class="QToolButton" name="downloadDirToolButton" >
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QCheckBox" name="closeDialogCheckBox" >
      <property name="text" >
       <string>Close file transfer dialog after finish</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2" >
     <widget class="QLineEdit" name="downloadDirLineEdit" />
    </item>
    <item row="3" column="0" colspan="3" >
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Automatic file receiving</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="receiveFromListRadioButton" >
         <property name="text" >
          <string>Receive files from people on your list</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QRadioButton" name="alwaysAskRadioButton" >
         <property name="text" >
          <string>Always ask for permission</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Default download directory:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>Sending port:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>useProxyCheckBox</tabstop>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>portLineEdit</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>useProxyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>200</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel" >
     <x>200</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
